Sources/TCPConnection.swift

```diff
@@ -23,6 +23,11 @@
 // SOFTWARE.
 
 import CLibvenice
+import protocol C7.Connection
```

Comment:
Rather than importing the specific dependencies, I think it would be cleaner to just import C7.